op-deployer/book/src/reference-guide/custom-superchains.md

```diff
@@ -0,0 +1,157 @@
+# Custom Superchains
+
+While OP Deployer was designed primarily for use with the Optimism Superchain, it also supports managing chains that
+were deployed to custom Superchain deployments. This is particularly common for RaaS providers, whose customers often
+request deployments with custom L1s (or L2s, in the case of L3s) or governance. This guide will walk you through the
+process of managing these chains using OP Deployer.
+
+> [!danger]
+> Chains deployed in this way are not subject to Optimism Governance. They may be running customized or unaudited
+> code. Use at your own risk.
+
+## Bootstrapping
+
+The first step to deploying a custom Superchain is to bootstrap it onto an L1. This process will:
+
+- Deploy contract implementations that will be shared among all OP Chains deployed using this Superchain.
+- Deploy Superchain-wide management contracts like `SuperchainConfig` and `SuperchainVersions`.
+- Set up ownership so that you can control the Superchain.
+
+You will use the `bootstrap` [family][bootstrap] of commands on `op-deployer` to do this.
+
+[bootstrap]: ../user-guide/bootstrap.md
+
+### Superchain Bootstrap
+
+To begin, bootstrap the Superchain onto your chosen L1 with the following command:
+
+```shell
+op-deployer bootstrap superchain \
+  --l1-rpc-url="<rpc url>" \
+  --private-key="<contract deployer private key>" \
+  --artifacts-locator="<locator>" \
+  --outfile="<path to outfile>" \
+  --superchain-proxy-admin-owner="<role address>" \
+  --protocol-versions-owner="<role address>" \
+  --guardian="<role address>"
+```
+
+This will output a JSON file containing the addresses of the relevant contract. Keep track of this file, as you will
+need it in subsequent steps.
+
+We recommend the following these best practices when bootstrapping the Superchain:
+
+1. Use Gnosis SAFEs for ownership roles like `guardian` and `superchain-proxy-admin-owner`. The owner **must** be a
+   smart contract to support future upgrades, so a SAFE is a sensible default.
+2. Use a regular EOA as the deployer. It will not have any control over the Superchain once the deployment completes.
+3. Use a standard contracts tag (e.g. `tag://op-contracts/v2.0.0`). This will make upgrading easier.
```

Comment:
what's the best way for users to find the _latest_ standard contracts tag? we've had some confused feedback on this recently, maybe we can add a bit of detail on this here.

Reply:
i can add some color on this

Reply:
This will be fixed separately.